bgbackup.sh

```diff
@@ -502,7 +504,7 @@ fi
 mysqlcreate
 
 # Check that mysql client can connect
-$mysqlcommand "SELECT 1 FROM DUAL"
+$mysqlcommand "SELECT 1 FROM DUAL" 1>/dev/null
```

Comment:
Did you test this both with correct info and incorrect to verify it works both ways as expected?